epitome/test/__init__.py

```diff
@@ -4,19 +4,18 @@
 import unittest
 from epitome.models import *
 from epitome.dataset import *
-
-# set Epitome data path to test data files for testing
-# this data was saved using functions.saveToyData(<epitome_repo_path>/epitome/test/daata)
-dir_path = os.path.dirname(os.path.realpath(__file__))
-os.environ["EPITOME_DATA_PATH"] = os.path.abspath(os.path.join(dir_path, "data","test"))
-
-S3_TEST_PATH = 'https://epitome-data.s3-us-west-1.amazonaws.com/test.zip'
+from epitome.functions import download_and_unzip
 
 class EpitomeTestCase(unittest.TestCase):
 
 	def __init__(self, *args, **kwargs):
-		# download test data to parent dir of EPITOME_DATA_PATH  if it was not yet downloaded
-		download_and_unzip(S3_TEST_PATH, os.path.dirname(os.environ["EPITOME_DATA_PATH"]))
+		# class definitions
+		self.S3_DATA_PATH = 'https://epitome-data.s3-us-west-1.amazonaws.com/test.zip'
```

Comment:
why doe we need this information? Shouldn't this just be automatically downloaded in the new implementation by providing assembly name of `test`?

Reply:
This is being used to download the test dataset without relying on the Dataset's get_data_dir() function. Do you think we should create a global EpitomeDataset in the init function? I'm wondering if this may lead to dependencies on the EpitomeDataset for other functions?

Reply:
Why is this? Can't you just set `assembly="test"` and `data_dir=self.epitome_test_path` and it should automatically work?

Reply:
Are we updating this based on the previous comments?

Reply:
Done